rma/views/rma_views.xml

```diff
@@ -9,8 +9,25 @@
         <field name="arch" type="xml">
             <search>
                 <field name="name" />
+                <field name="origin" />
                 <field name="user_id" />
                 <field name="tag_ids" />
+                <filter
```

Comment:
Shouldn't a separator be here?

Reply:
@sbejaoui

Reply:
@pedrobaeza , no need for a separator between the search fields and filters

![image](https://github.com/user-attachments/assets/8d329a68-5adb-4d33-bc43-574ad2a8cfe9)